charts/unikorn/templates/clustermanagerapplicationbundles.yaml

```diff
@@ -9,7 +9,30 @@ spec:
     reference:
       kind: HelmApplication
       name: vcluster
-      version: 0.15.7
+      version: 0.19.5
+  - name: cert-manager
+    reference:
+      kind: HelmApplication
+      name: cert-manager
+      version: v1.12.4
+  - name: cluster-api
+    reference:
+      kind: HelmApplication
+      name: cluster-api
+      version: v0.1.11
+---
+apiVersion: unikorn-cloud.org/v1alpha1
+kind: ClusterManagerApplicationBundle
+metadata:
+  name: cluster-manager-1.1.0
+spec:
+  version: 1.1.0
```

Comment:
Given that:

* Nothing is actually deployed yet and no one is using it
* This is not marked as preview, so everyone will be upgraded anyway

Isn't this a bit pointless?

Reply:
Yeah fair point, didn't want to make any assumptions given there are development environments out there and I basically forgot about the preview option 🤦

My plan was to fold in a bunch more updates (cert-manager etc.) into version 1.1.0 and it becomes the new default so can mark this preview for now, happy to go with whatever strategy you think is best though?